app/controllers/api/v1/orders_controller.rb

```diff
@@ -12,9 +13,9 @@ class Api::V1::OrdersController < ApiController
   # t.datetime "updated_at",       null: false
 
   def create
-    attributes = order_params
+    attributes = order_params.merge(user_id: current_user.id) if current_user
```

Comment:
is `if current_user` necessary here? 
looking at `authenticate_api_user` method, it is instantly returning 403 if no user is found. We should always have a `current_user` at this point.